### PR TITLE
#341 - Change Heading's static values to variables from theme.js

### PR DESCRIPTION
**Pull request by snowiesuet**
Sorry for the multiple pull requests, there was a merge issue!

### CODE REVIEW

File: client/src/components/Feed/FilterBox.js [hunk 1]
```diff
@@ -42,7 +42,7 @@ const FilterBox = () => {
   return (
     <FilterBoxWrapper className="filter-box">
       <TextLabel
-        block={true}
+        block={"true"}
```
Comment (anniegiang) [inline client/src/components/Feed/FilterBox.js:45]: Let's remove the curly brackets, `block="true"` is fine

File: client/src/components/Typography/Heading.js [hunk 2]
```diff
@@ -14,22 +14,22 @@ const Heading = styled(Title)`
   }
 
   &.h1 {
-    font-size: 4.2rem;
+    font-size: ${theme.typography.heading.one};
```
Comment (anniegiang) [inline client/src/components/Typography/Heading.js:17]: Let's destructuring `heading` like this:

`const { heading } = theme.typography`

and then replace wherever it's use, for example like this `heading.one`

File: client/src/pages/Profile.js [hunk 3]
```diff
@@ -174,11 +174,15 @@ const Profile = (props) => {
       <div style={userInfoStyle}>
         {popover(props)}
         <ProfilePic noPic={true} initials={getInitials(firstName, lastName)} />
-        <TextLabel weight="500" block={true} size={theme.typography.size.large}>
+        <TextLabel
+          weight="500"
+          block={"true"}
```
Comment (anniegiang) [inline client/src/pages/Profile.js:179]: Let's remove the curly brackets as well

File: client/src/components/Typography/Heading.js (hunk 2 above)
Comment (snowiesuet) [reply client/src/components/Typography/Heading.js:17]: ah thanks! i was wondering how we can shorten it

**Pull request merged**